lib/ex_subtil_backend/amqp/job_acs_error_consumer.ex

```diff
@@ -8,6 +8,12 @@ defmodule ExSubtilBackend.Amqp.JobAcsErrorConsumer do
     consumer: &ExSubtilBackend.Amqp.JobAcsErrorConsumer.consume/4
   }
 
+  def consume(channel, tag, _redelivered, %{"job_id" => job_id, "error" => description} = payload) do
+    Logger.error("Command line error #{inspect(payload)}")
+    Status.set_job_status(job_id, "error", %{"code": nil, "message": description})
```

Comment:
Why set `code` to nil ?
if you can't get it don't add key. If you can get it, you can store it ;-)